### ISSUE TITLE
When the comma is just a comma....

### ISSUE THREAD

**Issue by DianeNox** [scott2000/answerset#28]
Hi, again! I have toootally been staying on top of my Norwegian practice recently.....

Yeah who am I kidding. But I'm back now and hopefully will be able to make the habit stick again. And meanwhile, this has cropped up again. Basically, the entire plugin uses the commas as signifiers, right? But sometimes, a comma really is just that. A comma. And, like anyone typing things fast and without a care in the world, when the comma isn't needed I have a habit of dropping it.
Long story short: 
<img width="793" alt="Screenshot 2024-03-13 130906" src="https://github.com/scott2000/answerset/assets/51899303/962e4259-d645-4e30-b541-e29906f32063">

Would it be possible to add a quick "if" check before the plugin truly kicks in, that checks for this situation?

**Comment by scott2000** [comment]
I think this should be possible to add, let me look into it. In the meantime, you could add a semicolon `;` at the end of the correct answer. The way it actually works is that it checks if there's a semicolon, and if so it uses semicolons as the separator, otherwise it uses commas. So if you add a semicolon at the end of the correct answer, it will tell it to use semicolons and ignore commas for that particular answer.

**Comment by scott2000** [comment]
This should be working with the latest update! Here is how it would look now:

<img width="573" alt="After Update" src="https://github.com/scott2000/answerset/assets/5590720/aa51234f-d41a-4848-9bea-cbbb9209749e">

If you add comma to the "Ignored Characters" in the config, you could even get it to not show any error. There is also the new "Separators" config option as suggested in #31.
